libs/pages/elewa/news-detail/src/lib/pages/news-related-articles/news-related-articles.component.html

```diff
@@ -0,0 +1,26 @@
+<h1>Related articles</h1>
+<div class="container">
+  <div class="card">
+    <img src="https://res.cloudinary.com/db15gy9h6/image/upload/v1678350142/Screenshot_from_2023-03-09_11-19-16_r7nfrz.png" alt="" srcset="">
+    <p>Introducing Conversational learning ipsum dolar</p>
+    <span class="card-date">20 Oct 2022</span>
+    <span class="dot">&#183;</span>
+    <span class="card-timestamp">5 min read</span>
+  </div>
+  <div class="card">
+    <img src="https://res.cloudinary.com/db15gy9h6/image/upload/v1678350142/Screenshot_from_2023-03-09_11-19-43_ihgsdr.png" alt="" srcset="">
+    <p>Introducing Conversational learning ipsum dolar</p>
+    <span class="card-date">20 Oct 2022</span>
+    <span class="dot">&#183;</span>
+    <span class="card-timestamp">5 min read</span>
+  </div>
+  <div class="card">
+    <img src="https://res.cloudinary.com/db15gy9h6/image/upload/v1678350142/Screenshot_from_2023-03-09_11-19-28_ct81ae.png" alt="" srcset="">
+    <p>Introducing Conversational learning ipsum dolar</p>
+    <span class="card-date">20 Oct 2022</span>
+    <span class="dot">&#183;</span>
+    <span class="card-timestamp">5 min read</span>
+  </div>
+  <div></div>
+  <div></div>
```

Comment:
Use a list from the component here. Put the articles in a list and loop over them on the html